### DISCUSSION TITLE
Recommended/favorite chrome downgrade version?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1057] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
If you have downgraded, and let's say someone was doing the same and wondered what version to downgrade to, what would you say?

Later edit: sorry for the limited options. Github won't let me add more.

**Comment by AnOrangeCone** [comment]
i always go 114 since its the oldest you can really go with kernel version v2

**Reply by FunnyCorgi** [reply]
Can you go older with v2?

**Reply by AnOrangeCone** [reply]
i dont think so i tried but there was no downloads you should be able to go to v112 but im unsure since i cant find a v112 for octopus phaser360

**Reply by AnOrangeCone** [reply]
but on v114 you can use cryptosmite (fakemurk after if you want I DONT RECOMMEND JUST RE ENROLL WHEN NEEDED)

**Comment by AshtonDavies** [comment]
The best Chrome OS version of all time for exploits is v105. It supports almost every single exploit that has been discovered.

**Reply by FunnyCorgi** [reply]
@AshtonDavies What kernel version does it support? Can it be downgraded to from 121 on kernel v2?

**Reply by AshtonDavies** [reply]
> @AshtonDavies What kernel version does it support? Can it be downgraded to from 121 on kernel v2?

I think several versions released since 2010 that are below v120 or v121 are at kernel v2, which includes v105.

They changed the base from good old Gentoo Linux, which matched Android, to the more popular Debian.

**Comment by CGRmags** [comment]
just saying i perfer 120 bc of skiovox and you can use your school browser to

**Reply by FunnyCorgi** [reply]
Which method do you use?
